components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -1749,6 +1756,39 @@ this mask will have smb calculated over the entire global land surface
+<map frm_hgrid="10x10min"    frm_lmask="nomask"  to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_10x10min_nomask_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="3x3min"    frm_lmask="MODIS"  to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_3x3min_MODIS_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="3x3min"    frm_lmask="USGS"   to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_3x3min_USGS_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="3x3min"    frm_lmask="LandScan2004" to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_3x3min_LandScan2004_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="5x5min"    frm_lmask="IGBP-GSDP"  to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_5x5min_IGBP-GSDP_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="5x5min"    frm_lmask="ISRIC-WISE"  to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_5x5min_ISRIC-WISE_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="5x5min"    frm_lmask="nomask"  to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_5x5min_nomask_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="3x3min"   frm_lmask="GLOBE-Gardner" to_hgrid="northamericax4v1pg2" to_lmask="nomask"
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_3x3min_GLOBE-Gardner_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="3x3min"   frm_lmask="GLOBE-Gardner-mergeGIS" to_hgrid="northamericax4v1pg2" to_lmask="nomask"
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_3x3min_GLOBE-Gardner-mergeGIS_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="0.9x1.25"    frm_lmask="GRDC"  to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_0.9x1.25_GRDC_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="360x720cru"    frm_lmask="cruncep"  to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_360x720cru_cruncep_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="1km-merge-10min"    frm_lmask="HYDRO1K-merge-nomask"  to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_1km-merge-10min_HYDRO1K-merge-nomask_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
+<map frm_hgrid="0.5x0.5"    frm_lmask="GSDTG2000"  to_hgrid="northamericax4v1pg2"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/northamericax4v1pg2/map_0.5x0.5_GSDTG2000_to_northamericax4v1pg2_nomask_aave_da_c210112.nc</map>
```

Comment:
@bishtgautam , I am not sure if the `to_hgrid` field needs to be renamed or not. It seems to me this session only affects the maps in the preprocessing step, instead of the model simulation. I didn't change it in my bi-grid RRM test, which ran successfully. Change it or leave it as you see fit.